pom.xml

```diff
@@ -1583,6 +1585,7 @@
                         </dependency>
                     </dependencies>
 
+                    <!-- // @todo: Why is the execution configure here and not here: https://github.com/mulesoft/mule-ee-distributions/blob/e6370956039d68d2f9f2bfea00dab46bae685ad8/standalone/pom.xml#L138-->
```

Comment:
It should be configured both in mule-distributions and mule-ee-distributions

Reply:
I agree on that the execution should be configured in the Maven module where it's used, but I wonder if this change could break other unknown to us usages, let's say in [`tooling`](https://github.com/mulesoft/mule-tooling-api/blob/master/pom.xml#L441-L445). BTW: I haven't check in `tooling` has an ancestry relation with `mule`

Reply:
Ale, I just found that the execution configuration was moved from `mule-distributions` 14 days ago [take a look](https://github.com/mulesoft/mule-distributions/pull/629/files) to `mule` [take a look](https://github.com/mulesoft/mule/pull/10470/files) and also the groovy version override was part of that change.

The change we've been talking about is somehow the rollback of that change. Might I be missing something? Any idea on which was the motivation of [MULE-19578](https://www.mulesoft.org/jira/browse/MULE-19578) ?

Reply:
Yeah, it seems that something is wrong there. It should be moved back to mule-distributions and mule-ee-distributions. Because `pluginManagement` is meant only for version handling.